packages/card/components/Card.tsx

```diff
@@ -70,7 +70,9 @@ class Card<P extends CardProps, S extends {}> extends React.PureComponent<
             <img src={header.headerImg} alt={header.headerImgAltText ?? ""} />
           </div>
         )}
-        <div className={padding("all", paddingSize)}>{children}</div>
+        <div className={cx(padding("all", paddingSize), cardContent)}>
```

Comment:
this may be a breaking change. Or perhaps we can call it a bug fix

Reply:
I put that bit under the first fix commit. I don't believe it should be a breaking change, as it's retaining the previous style of height 100% but making sure that it does not get applied to the card header div I added. I wasn't able to see this issue until passing in an image with a set height and width from placehold.it